### PR TITLE
Update message center setup

### PR DESCRIPTION
**Pull request by alexiusko**

### CODE REVIEW

File: src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java [hunk 1]
```diff
@@ -27,53 +29,66 @@
...
-                .as(DraftsPostResponseBody.class)
-                .getId();
+                .path("id");
+        topicIds.add(topicId);
+
+        IntStream.range(0, randomAmount).forEach(randomNumber -> {
+
+            loginRestClient.login(externalUserId, externalUserId);
+
+            DraftsPostRequestBody draftsPostRequestBody = MessagesDataGenerator.generateDraftsPostRequestBody(topicIds);
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java]: was it intentional having only one element in the list of topic ids? you might want to remove the list wrapper and pass on the bare topicId

File: src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java [hunk 2]
```diff
@@ -27,53 +29,66 @@
...
     private final LoginRestClient loginRestClient;
     private final MessagesPresentationRestClient messagesPresentationRestClient;
     private String rootEntitlementsAdmin = globalProperties.getString(PROPERTY_ROOT_ENTITLEMENTS_ADMIN);
 
     public void ingestConversations(String externalUserId) {
         int randomAmount = CommonHelpers
-            .generateRandomNumberInRange(globalProperties.getInt(PROPERTY_MESSAGES_MIN),
-                globalProperties.getInt(PROPERTY_MESSAGES_MAX));
-        IntStream.range(0, randomAmount).forEach(randomNumber -> {
+                .generateRandomNumberInRange(globalProperties.getInt(PROPERTY_MESSAGES_MIN),
```
Comment (richard-blank) [inline src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java]: is it  possible to undo the additional indenting?

Comment (alexiusko) [reply src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java]: yes

File: src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java (hunk 1 above)
Comment (alexiusko) [reply src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java]: yes, it was intentionally. I think someone will want to have more topics for testing.

**Comment by alexiusko** [comment]
image
Topics were created successfully.

File: src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java (hunk 1 above)
Comment (alexiusko) [reply src/main/java/com/backbase/ct/bbfuel/configurator/MessagesConfigurator.java]: updated PR

**Pull request merged**